### PR TITLE
dlgPlotregion

### PR DESCRIPTION
**Pull request by shadrackkibet**
@dannyparsons  here is the dialog. Kindly have a look at the icalltype issue. I have not worked on Testok and  the designer properly.

### CODE REVIEW

**Comment by shadrackkibet** [comment]
@africanmathsinitiative/developers this is ready for testing.

File: instat/dlgPlotRegion.vb [hunk 1]
```diff
@@ -50,24 +52,26 @@ Public Class dlgPlotRegion
...
         ucrReceiverLongitude.SetMeAsReceiver()
-        clsPlotRegionFunction.AddParameter("time_point", clsRFunctionParameter:=clsAsDateTimeFunction)
+        clsPlotRegionFunction.AddParameter("time_point", clsRFunctionParameter:=clsAsDateTimeFunction, iPosition:=5)
 
         clsPlotRegionFunction.SetRCommand("plot.region")
         clsAsDateTimeFunction.SetRCommand("as.Date")
+        clsSequenceFunction.SetRCommand("seq")
         clsAsDateTimeFunction.AddParameter("format", Chr(34) & "%Y-%m-%d" & Chr(34), iPosition:=1)
-        dtpDateTimePicker.Value = New DateTime(2015, 1, 1)
+        dtpDateTimePicker.Value = New DateTime(2015, 11, 1)
```
Comment (dannyparsons) [inline instat/dlgPlotRegion.vb]: Can you put this back to 1? Neither are sensible really but at least starting January looks better for now.

Comment (shadrackkibet) [reply instat/dlgPlotRegion.vb]: ok

Comment (shadrackkibet) [reply instat/dlgPlotRegion.vb]: wanted this to be easy for the data i am using to test but forgot to change this!

Comment (shadrackkibet) [reply instat/dlgPlotRegion.vb]: can this be 2015,01,01 ?

Comment (dannyparsons) [reply instat/dlgPlotRegion.vb]: yes as it was before is fine

**Pull request merged**